### DISCUSSION TITLE
how to determine the risk for a trait or disease from the Aggregated SUM

### DISCUSSION THREAD

**Discussion by Asakura12** [PGScatalog/pgsc_calc#227] [Q&A] [answered]
**sampleset	IID	        PGS	                SUM	        DENOM	AVG**
ColonCancer	TRUTH	PGS000376	0.0440565	1140256	3.86373761681587e-08
ColonCancer	QUERY	PGS000376	0.0437259	1140236	3.83481138992279e-08

From the Table Above, how to determine the risk for a trait or disease from the given data?

**Comparative Analysis**: Compare the SUM values between the TRUTH and QUERY samples for the same PGS (PGS000376). A higher SUM value might suggest a relatively higher genetic predisposition to the trait or disease in the sample with the higher SUM. But, how do we set a threshold for this to categorize individuals into high, medium, or low-risk groups.

Or am I taking an incorrect approach?

**Comment by Fiwx** [comment, marked as answer]
- How do we set a threshold for this to categorize individuals into high, medium, or low-risk groups?
We do a study, and see which percentile of PRS corresponds to a low, medium, or high-risk.

- From the Table Above, how to determine the risk for a trait or disease from the given data?
You can reference an earlier study, use a background distribution of many genomes to create a percentile distribution, and then see which percentile that sum lies in. It is not that useful by itself. (Note: you are assuming that 90th percentile PRS in your study is the same sum as 90th percentile in your background distribution. This is not always true, especially when data is sourced from places like hospitals.)
